src/db_queries.js

```diff
@@ -15,6 +15,31 @@ dbQueries.getPostData = (connPool, id, callback) => {
   );
 };
 
+dbQueries.getTags = (connPool, id, callback) => {
+  connPool.query(
+    'SELECT tag_id FROM posts_tags WHERE post_id = $1',
+    [id],
+    callback
+  );
+};
+
+dbQueries.getTagNames = (connPool, tagIds, callback) => {
+  let queryString = 'SELECT description FROM tags WHERE ';
+  const dataArray = [];
```

Comment:
then you don't need to push to the dataArray, you just tell the callback to return those elements that pass the if statement